highbury/assetlist.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "highbury",
+  "assets": [
+    {
+      "description": "The native staking and governance token of the Blackfury ecosystem",
+      "denom_units": [
+        {
+          "denom": "ufury",
+          "exponent": 0
+        },
+        {
+          "denom": "fury",
+          "exponent": 6
+        }
+      ],
+      "base": "ufury",
+      "name": "Fury",
+      "display": "fury",
+      "symbol": "FURY",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.png"
+      },
+      "coingecko_id": "fury"
+    },
+    {
+      "description": "Governance token of Jinx Wallet and Fury Lend Protocol",
+      "denom_units": [
+        {
+          "denom": "jinx",
+          "exponent": 0
+        },
+        {
+          "denom": "JINX",
+          "exponent": 6
+        }
+      ],
+      "base": "jinx",
+      "name": "Jinx",
+      "display": "JINX",
+      "symbol": "JINX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinx.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinx.svg"
+      },
+      "coingecko_id": "jinx"
+    },
+    {
+      "description": "Governance token of Fury Swap Protocol",
+      "denom_units": [
+        {
+          "denom": "jinxy",
+          "exponent": 0
+        },
+        {
+          "denom": "JINXY",
+          "exponent": 6
+        }
+      ],
+      "base": "jinxy",
+      "name": "Jinxy",
+      "display": "JINXy",
+      "symbol": "JINXy",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinxy.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinxy.svg"
+      },
+      "coingecko_id": "fury-swap"
+    },
+    {
+      "description": "The native stablecoin of Fury",
+      "denom_units": [
+        {
+          "denom": "usdf",
+          "exponent": 0
+        },
+        {
+          "denom": "USDF",
+          "exponent": 6
+        }
+      ],
+      "base": "usdf",
+      "name": "USDF",
+      "display": "USDF",
+      "symbol": "USDF",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.png"
+      },
+      "coingecko_id": "usdf"
+    },
+    {
+      "denom_units": [
```

Comment:
Pleas remove this asset. I'm guessing it was left here from a copy-paste??

Reply:
Irrelevant asset removed.

Reply:
I'm referring to this USDT--it's still here...

Reply:
That's removed too. Please proceed with your review. Thank you.